src/styles/components/common/partners/partner_block.scss

```diff
@@ -14,4 +14,11 @@
   .description {
     margin-bottom: 35px;
   }
+
+  .c-btn {
```

Comment:
We should try to avoid this kind of css, we should never add a component class inside another component. Use another class or the tag name